VicenteDouriet/ChallengesDay26/Hackerrank/SolutionInheritance2.java

```diff
@@ -0,0 +1,27 @@
+class Arithmetic
+{
+  private int a;
+  private int b;
+  
+  void add(int a,int b) {
+    System.out.print(a+b+" ");
+    
+  }
+}
+class Adder extends Arithmetic {
+    Adder(){ 
+      System.out.println("My superclass is: Arithmetic");
```

Comment:
this is hardcoded, what if I change the Arithmetic class name?
check 
getClass().getSimpleName()